catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -295,6 +295,7 @@
                             ddf.catalog.filter.impl,
                             ddf.catalog.history,
                             ddf.catalog.impl,
+                            ddf.catalog.impl.operations,
```

Comment:
Did you actually need this?  I am not sure why this project is private packaging its own packages.